scripts/PerfHarness/PerfHarness.cs

```diff
@@ -30,6 +30,7 @@ private static string[] GetTestAssemblies()
             "Benchmarks",
             "System.Binary.Base64.Tests",
             "System.Text.Primitives.Tests",
+            "System.Text.Primitives.Encoding.Performance.Tests"
```

Comment:
Why a new project? I think you should just add it to System.Text.Primitives.Tests